Source/WebKit/UIProcess/API/Cocoa/WKDownload.h

```diff
@@ -34,6 +34,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 WK_CLASS_AVAILABLE(macos(11.3), ios(14.5))
+WK_SWIFT_UI_ACTOR
```

Comment:
does this have to go through API review?

Reply:
API review is ✅.